backend/mobile-backend/schema.graphql

```diff
@@ -0,0 +1,58 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } } # FOR TESTING ONLY!
+
+type GameSession @model {
+  id: ID! @primaryKey(sortKeyFields: ["updatedAt"])
+  game: Game! @hasOne
+  startTime: AWSDateTime
+  phaseOneTime: Int
+  phaseTwoTime: Int
+  teams: [Team] @hasMany
+  updatedAt: AWSDateTime!
+}
+
+type Game @model {
+  id: ID!
+  title: String
+  description: String
+  imageUrl: String
+  isAdvanced: Boolean
+  gameCode: Int
+  grade: String
+  questions: [Question] @hasMany
+}
+
+type Question @model {
+  id: ID!
+  question: String!
+  answer: String
+  trickAnswers: AWSJSON
+  imageUrl: String
+  instructions: AWSJSON
+  grade: String
+}
+
+type Team @model(subscriptions: null){
+  id: ID!
+  name: String!
+  gameId: ID!
+  questionId: ID!
+  trickiestAnswerID: ID!
+  teamMembers: [TeamMember] @hasMany
+}
+
+type TeamMember @model(subscriptions: null) {
```

Comment:
I think we'd need to enable `subscription` for this. When game starts, we want to know when each member joins the game

Reply:
Yes makes sense! I'll add that subscription in